### PR TITLE
Fixed dependencies, removed Sys. println

### PR DESCRIPTION
**Pull request by mskacelik**
/cc @jmartisk

### CODE REVIEW

File: client/model-builder/src/main/java/io/smallrye/graphql/client/model/SmallRyeGraphQLClientMessages.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+import org.jboss.logging.Messages;
+import org.jboss.logging.annotations.Cause;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageBundle;
+
+import io.smallrye.graphql.client.InvalidResponseException;
+
+@MessageBundle(projectCode = "SRGQLDC")
+public interface SmallRyeGraphQLClientMessages {
```
Comment (jmartisk) [inline client/model-builder/src/main/java/io/smallrye/graphql/client/model/SmallRyeGraphQLClientMessages.java:14]: I don't understand the purpose of this. These are errors that happen at runtime, but you add this to a module that is only present at build time (and you're also leaving the original copy in the `client/implementation` module which means we would have this duplicated

Comment (mskacelik) [reply client/model-builder/src/main/java/io/smallrye/graphql/client/model/SmallRyeGraphQLClientMessages.java:14]: The idea was that I did not want to have a dependency on `implementation` in the `model-builder.`
We use one of the messages for recursion exception during query building.

Comment (jmartisk) [reply client/model-builder/src/main/java/io/smallrye/graphql/client/model/SmallRyeGraphQLClientMessages.java:14]: Then we should have this class only contain the message relevant to recursion, and change the numbers to not overlap with the original logging class. Let's switch from 35000-39999 to 40000-44999, that range isn't used right now.And the message about recursion will have an ID of 40000

**Comment by mskacelik** [comment]
new quarkus branch: https://github.com/mskacelik/quarkus/tree/srgql-2.8.4

**Review by jmartisk — APPROVED** [review]

**Pull request merged**